src/board-explorer-app/board-explorer-app.html

```diff
@@ -968,7 +972,7 @@
       selected.forEach(function(refdes) {
         /* Disable the button that matches this refdes */
         Array.prototype.forEach.call(this.$.layout.querySelectorAll("paper-button[match='"+ refdes +"']"), function(el) {
-          el.disabled = true;
+          el.disabled = false;
         });
```

Comment:
Is this change intentional? If so, the comment should be changed to indicate that the code is enabling the button that matches the refdes.